python/cudf/cudf/dataframe/categorical.py

```diff
@@ -64,7 +64,11 @@ def as_unordered(self, inplace=False):
             return Series(data=self._parent.replace(ordered=False))
 
     def add_categories(self, new_categories, **kwargs):
-        data = None if kwargs["inplace"] else self._parent
+        inplace = False
```

Comment:
Apologies for the key error here. We could make this a one-liner again with `kwargs.get('inplace', False)` instead of `kwargs['inplace']`.